types/datejs/index.d.ts

```diff
@@ -8,17 +8,6 @@
 
 /// <reference path="sugarpak.d.ts" />
 
-/** Defines a structure to be used in the add and set instance methods */
```

Comment:
Please can we keep this interface

Reply:
The interface was removed because a) it's only ever used as the argument to add (set uses different names) and b) it's only ever useful as a partial - I didn't see any compelling reason to leave it in